cloudformation/ecs-selenium.cfn.yml

```diff
@@ -596,6 +625,10 @@ Resources:
           ImageId: !FindInMap [AMIRegionMap, !Ref "AWS::Region", "64"]
           InstanceType: !Ref NodeInstanceType
           KeyName: !Ref KeyName
+          BlockDeviceMappings:
+            - DeviceName: "/dev/xvdcz"
```

Comment:
I think this device name will be different for NVME-based instance types (c5, etc). You might need to set up a mapping to accommodate them or simply allow the user to pass it as a parameter that defaults to what you have here.

Same goes for the same property below (line 670)